src/components/data-management/ProjectsList.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Space } from 'antd';
+
+import ProjectCard from './ProjectCard';
+
+const ProjectsListContainer = (props) => {
```

Comment:
rename this to `ProjectsList` -- the components names have to match the file names.

Reply:
Just noticed that we have this `ProjectsList` component defined both under `ProjectsListContainer.jsx` and `ProjectsList.jsx` files. This must be wrong, make sure you delete whichever of them is not used.

Reply:
Whoops, my bad. fixed